File: Chapter0_Algorithm/2402/D240228/bj_1929.py

import math

def is_prime_number(x) :
    result = False
    for i in range(2, int(math.sqrt(x))+1) :
        if(x%i == 0) :
            result = True
            break
    if x == 1 :
        return True
    return result

n, m = map(int, input().split(" "))

for i in range(n, m+1) :
    if(not is_prime_number(i)):
        print(i)

File: Chapter0_Algorithm/2402/D240229/bj_1929.java
package D240229;

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class bj_1929 {
    public static void main(String args[]) throws Exception {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        StringTokenizer st = new StringTokenizer(br.readLine());
        int M = Integer.parseInt(st.nextToken());
        int N = Integer.parseInt(st.nextToken());

        for (int i = M; i <= N; i++) {
            if (is_prime_number(i)) {
                System.out.println(i);
            }
        }
    }

    private static boolean is_prime_number(int n) {
        boolean result = true;
        if (n == 1) {
            return false;
        }
        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) {
                result = false;
                break;
            }
        }
        return result;
    }
}